bucket/ccl.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "1.11.5",
+    "homepage": "https://ccl.clozure.com/",
+    "description": "Clozure Common Lisp",
+    "license": "Apache-2.0",
+    "url": "https://github.com/Clozure/ccl/releases/download/v1.11.5/ccl-1.11.5-windowsx86.zip",
+    "hash": "da5d8830e38742a5e76dc09a5fdebf8cb8a09a6fee87d7919ffd67ae2584cdd4",
+    "extract_dir": "ccl",
+    "env_add_path": "scripts",
+    "bin": [
+        "wx86cl.exe",
+        "wx86cl64.exe"
```

Comment:
What is different in these files??
Is it used to produce different compiled binary when talking about architectures? Like 64.exe produce only 64bit output?

Reply:
> Like 64.exe produce only 64bit output?

Yes, exactly.

They are exactly the same, beside one is 64-bit, the other 32-bit. The difference is that the Lisp developer may develop an application with 64-bit, and deploy (ie create an exe file) both 64-bit and 32-bit. But to create one or the other one needs the proper corresponding binary.

Reply:
@priyadarshan Thanks.

@linsui Keep it as it is